src/content/changes/workload-cluster-releases-aws/releases/aws-v20.0.0.md

```diff
@@ -0,0 +1,18 @@
+---
+# Generated by scripts/aggregate-changelogs. WARNING: Manual edits to this files will be overwritten.
+aliases:
+- /changes/tenant-cluster-releases-aws/releases/aws-v20.0.0/
+changes_categories:
+- Workload cluster releases for AWS
+changes_entry:
+  repository: giantswarm/releases
+  url: https://github.com/giantswarm/releases/tree/master/aws/v20.0.0
+  version: 20.0.0
+  version_tag: v20.0.0
+date: '2021-05-31T14:50:41+00:00'
+description: Release notes for AWS workload cluster release v20.0.0, published on
+  31 May 2021, 14:50
+title: Workload cluster release v20.0.0 for AWS
+---
+
+
```

Comment:
@alex-dabija Looks like v20.0.0 has been released without release notes.

https://github.com/giantswarm/releases/tree/master/aws/v20.0.0

Reply:
I would like to announce KVM 14.1.0 so I have removed this release from the PR.

Reply:
It appears that the auto-generation restored it after I manually deleted it. Should I delete it in a separate PR or is it okay as it is? https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v20.0.0/

Reply:
As long as the generator finds the release, it will create a changelog entry for it. So in order to get rid of it we must either remove the release in giantswarm/releases, or change the generator to omit deprecated releases.

My preferred solution would be to add a bit of info to the v20.0.0 README explaining what this release is about and that's it for Giant Swarm testing purposes only. Afetr all, customers can run into this release in one place or another, so there should be basic info about it.